sdk/core/azure-core/src/exception.cpp

```diff
@@ -28,4 +30,48 @@ namespace Azure { namespace Core {
     RawResponse = std::move(rawResponse);
   }
 
+  RequestFailedException::RequestFailedException(
+      std::unique_ptr<Azure::Core::Http::RawResponse>& rawResponse)
+      : std::runtime_error(getRawResponseField(rawResponse, "message"))
+  {
+    auto& headers = rawResponse->GetHeaders();
```

Comment:
```suggestion
    auto& headers = RawResponse->GetHeaders();
```

Reply:
ditto for others

Reply:
i'm lost RawResponse is a type i'm using a parameter called rawResponse

Reply:
`RawResponse` is a member variable.